### PR TITLE
Geowave 829

### PR DESCRIPTION
**Pull request by bmendell**
#829 #828 
Replaced the xuggler library with jcodec.  Updated files to support 4676 data ingest and display.

### CODE REVIEW

File: core/mapreduce/src/main/java/mil/nga/giat/geowave/mapreduce/output/GeoWaveOutputFormat.java [hunk 1]
```diff
@@ -56,8 +58,32 @@
 					persistentAdapterStore.addAdapter(a);
 				}
 			}
+
 			final IndexStore persistentIndexStore = GeoWaveStoreFinder.createIndexStore(configOptions);
 			final Index[] indices = JobContextIndexStore.getIndices(context);
+			StringBuilder sbDebug = new StringBuilder();
```
Comment (rfecher) [inline core/mapreduce/src/main/java/mil/nga/giat/geowave/mapreduce/output/GeoWaveOutputFormat.java]: I think we should wrap this with if(LOGGER.isDebugEnabled()){<build string>} so we're not running through building this string in production

File: extensions/formats/stanag4676/service/src/main/java/mil/nga/giat/geowave/types/stanag4676/service/rest/Stanag4676ImageryChipService.java [hunk 2]
```diff
@@ -44,12 +45,25 @@
...
+import org.jcodec.containers.mkv.muxer.MKVMuxerTrack;
+import org.jcodec.scale.AWTUtil;
+import org.jcodec.scale.RgbToYuv420p;
 
 import com.google.common.io.Files;
-import com.xuggle.mediatool.IMediaWriter;
-import com.xuggle.mediatool.ToolFactory;
-import com.xuggle.xuggler.ICodec;
+//import com.xuggle.mediatool.IMediaWriter;
+//import com.xuggle.mediatool.ToolFactory;
```
Comment (rfecher) [inline extensions/formats/stanag4676/service/src/main/java/mil/nga/giat/geowave/types/stanag4676/service/rest/Stanag4676ImageryChipService.java]: feel free to delete the various commented lines of code - we have revision history if we wanted to revisit xuggler

File: core/store/src/main/java/mil/nga/giat/geowave/core/store/GeoWaveStoreFinder.java [hunk 3]
```diff
@@ -111,9 +110,7 @@ public static SecondaryIndexDataStore createSecondaryIndexDataStore(
 	private static List<String> getMissingRequiredOptions(
 			final StoreFactoryFamilySpi factory,
 			final Map<String, String> configOptions ) {
-		final ConfigOption[] options = ConfigUtils.createConfigOptionsFromJCommander(factory
-				.getDataStoreFactory()
-				.createOptionsInstance());
+		final ConfigOption[] options = ConfigUtils.createConfigOptionsFromJCommander(factory.getDataStoreFactory().createOptionsInstance());
```
Comment (rfecher) [inline core/store/src/main/java/mil/nga/giat/geowave/core/store/GeoWaveStoreFinder.java]: are the new format changes matching running maven builds (if not the next commit will end up reformatted this again)?

**Comment by dcy2003** [comment]
Reviewed

**Pull request merged**